src/metrics/exit.rs

```diff
@@ -121,3 +140,27 @@ impl Exit for JavaCode {}
 impl Exit for GoCode {}
 impl Exit for CssCode {}
 impl Exit for HtmlCode {}
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use super::*;
+
+    #[test]
```

Comment:
Can you also add a "basic" test with a single function? And one with two functions which are not nested?